modules/browser/BrowserCapabilities.js

```diff
@@ -304,4 +284,45 @@ export default class BrowserCapabilities extends BrowserDetection {
     supportsGetDisplayMedia() {
         return navigator.getDisplayMedia !== undefined;
     }
+
+    /**
+     * Checks if the browser supports the "sdpSemantics" configuration option.
+     * https://webrtc.org/web-apis/chrome/unified-plan/
+     *
+     * @returns {boolean}
+     */
+    supportsSdpSemantics() {
+        return this.isChromiumBased() && this._getChromiumBasedVersion() >= 65;
+    }
+
+    /**
+     * Returns the version of a Chromium based browser.
+     *
+     * @returns {Number}
+     */
+    _getChromiumBasedVersion() {
```

Comment:
does this overlap at all with the other work you did for the chromium-based/version stuff?  i thought you had added something that did this

Reply:
I just extracted it out since I need to use it in 2 places now.

Reply:
can you send the latest version of jitsi meet